src/components/Person/Person.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import requiredIf from 'react-required-if';
+import './Person.scss';
+
+export function Person({ name, age, sex, isMarried, partnerName }) {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {name}
```

Comment:
```suggestion
        {`My name is ${name}`}
```
You can use JS expression